src/open_inwoner/accounts/tests/test_user_constraints.py

```diff
@@ -0,0 +1,121 @@
+from unittest import skip
+
+from django.db import IntegrityError, transaction
+from django.test import TestCase
+
+from open_inwoner.accounts.choices import LoginTypeChoices
+
+from .factories import UserFactory
+
+
+class UserIdentifierConstraintTests(TestCase):
+    def test_unique_email_when_login_default(self):
+        UserFactory(email="foo@example.com", login_type=LoginTypeChoices.default)
+        UserFactory(email="bar@example.com", login_type=LoginTypeChoices.default)
+
+        with self.assertRaises(IntegrityError):
+            UserFactory(email="foo@example.com", login_type=LoginTypeChoices.default)
+
+    def test_not_unique_email_when_login_digid_or_eherkenning(self):
```

Comment:
We currently have a test that asserts that DigiD users cannot register with a duplicate email that belongs to a non-DigiD user (https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/accounts/tests/test_auth.py#L1151). Is that still relevant (and more generally, the entire `DuplicateEmailRegistrationTest` class)?

Reply:
It does make me wonder why we relaxed the constraint for `email` though :thinking:

Reply:
In practice some Digid users are sharing the same email, like couples or caregivers helping someone else with their digital issues.